src/scenes/Partners/Detail/PartnerTypesCard.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import React, { FC } from 'react';
+import { PartnerDetailsFragment } from './PartnerDetail.generated';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  cardContent: {
+    flexGrow: 1,
+  },
+  cardActions: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    paddingRight: spacing(2),
+  },
+}));
+
+interface PartnerTypeCardProps {
+  partner?: PartnerDetailsFragment;
+  onEdit: () => void;
+  className?: string;
+}
+
+export const PartnerTypeCard: FC<PartnerTypeCardProps> = ({
+  partner,
+  className,
+  onEdit,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={className}>
```

Comment:
Our other similar cards are clickable using a `CardActionArea`. The "Edit" button is just there for a UI signal to the user

Reply:
Added `CardActionArea`